app/templates/events/view/tickets/index.hbs

```diff
@@ -22,7 +22,7 @@
         <th>{{t 'Order Status'}}</th>
         <th class="right aligned">{{t 'Total Tickets'}}</th>
         <th class="right aligned">{{t 'Total Orders'}}</th>
-        <th class="right aligned">{{t 'Total Sales'}} (US$)</th>
+        <th class="right aligned">{{t 'Total Sales ('}}{{currency-symbol model.eventDetail.paymentCurrency}}{{')'}} </th>
```

Comment:
`)` is just a bracket, `{{}}` are not required for it.
Just write this as 
```hbs
{t 'Total Sales'}} ({{currency-symbol model.eventDetail.paymentCurrency}})
```

Reply:
Alright